project-icp/bee-pollinator-app
# Add Splash Page

## Overview

Adds a splash view for users to land on, so they can get some context about the application. Once ready, they can click "Get started" to move to the drawing area.

### Demo

![image](https://cloud.githubusercontent.com/assets/1430060/20495902/fa9638f8-aff0-11e6-9494-bb40117f2a3e.png)

## Testing Instructions

 * Check out this branch
 * Bundle vendor scripts to include new image:

        ./scripts/bundle.sh --vendor

 * Open the home page [localhost:8000](http://localhost:8000/). You should see the splash screen.
 * Click "Get started". You should be taken to the draw view.
 * Drawing should work as expected.
 * Back / forward buttons should work as expected.
 * You should be able to go directly to the draw view at [localhost:8000/draw](http://localhost:8000/draw)

Connects #47
